pkg/api/aim/dao/models/image.go

```diff
@@ -0,0 +1,25 @@
+package models
+
+type ImageValues struct {
+	BlobURI string      `json:"blob_uri"`
+	Caption string      `json:"caption"`
+	Context interface{} `json:"context"`
+	Format  string      `json:"format"`
+	Height  interface{} `json:"height"`
+	Index   interface{} `json:"index"`
+	Key     string      `json:"key"`
+	SeqKey  string      `json:"seqKey"`
+	Name    string      `json:"name"`
+	Run     interface{} `json:"run"`
+	Step    int         `json:"step"`
+	Width   interface{} `json:"width"`
+}
+
+type Image struct {
```

Comment:
i like the way you modeled this, but our DB uses table/model `Artifact` for these attributes. Maybe we can merge, could be in a new ticket.